18:01:51.516 <user_data_mysql.add_data_to_user_base_item:45> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 41, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:06:04.334 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:06:40.671 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:10:01.571 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:10:31.766 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:11:18.634 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:28:28.983 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:30:44.551 <user_data_mysql.add_data_to_user_base_item:45> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 41, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:31:59.678 <user_data_mysql.add_data_to_user_base_item:45> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 41, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:32:41.108 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:33:17.976 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
18:33:49.127 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
20:32:39.286 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'NoneType' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 42, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'NoneType' object has no attribute 'add'
20:34:35.517 <user_data_mysql.add_data_to_user_base_item:46> [ERROR] 'SessionCM' object has no attribute 'add'
Traceback (most recent call last):
  File "user_data_mysql.py", line 41, in add_data_to_user_base_item
    db_session.add(new_item)
AttributeError: 'SessionCM' object has no attribute 'add'
